src/compo/viirs_l1bnc2ioda.py

```diff
@@ -158,6 +166,11 @@ def _read(self):
                 orbit_ad[:] = 1
             self.varAttrs['satelliteAscendingFlag', metaDataName]['description'] = '0=descending, 1=ascending'
 
+            # NASA VIIRS apply secant or divide reflectance by
+            # cosine of solar zenith angle to get true reflectance
+            sec_term = 1. / np.cos(self.solar_za * np.pi / 180.) if (
+                self.apply_secterm and self.solar_za != 90.) else 1.
```

Comment:
```suggestion
            sec_term = 1. / np.cos(self.solar_za * np.pi / 180.) if (self.solar_za != 90.) else 1.
```

actually the cleanest way readibility

Reply:
It won't work because solar_za is an array, not a single value.

Reply:
yes ... so that is more reason to break it down

please see the update this to me is the most readable and clear in intent